src/pwm0/decoder/mod.rs

```diff
@@ -77,7 +74,7 @@ impl LOADR {
             1 => LOADR::GROUPED,
             2 => LOADR::INDIVIDUAL,
             3 => LOADR::WAVEFORM,
-            i => LOADR::_Reserved(i),
+            _ => unreachable!(),
```

Comment:
Wouldn't this cause a runtime panic, as opposed to a handleable edge case?

Reply:
Yeah.  The codegen logic for this isn't new.  Ironically, the most recent change to that section blames to you splitting up some code in svd2rust :-p 

I think the reason this changed is because the bit width is now different; line 27126 of the svd changes the msb for `LOAD`.  With the revised bit width, it should be impossible to read back any other value so perhaps it is fine to panic?